src/polkadot_rpc_wrapper.py

```diff
@@ -34,19 +34,6 @@ def is_syncing(self) -> str:
         response_json = json.loads(response.text)
         return response_json['result']['isSyncing']
 
-    def is_validating(self) -> bool:
```

Comment:
This is the old function that was used for the `Validating` in the status. It was quite meaningless. It basically only tells if the node is running with the `--validator/--collator` option or not.